Documentation/CRDs/Cluster/ceph-cluster-crd.md

```diff
@@ -974,3 +975,15 @@ spec:
     user's responsibility.
 
 The operator does not unset any removed config options, it is the user's responsibility to unset or set the default value for each removed option manually using the Ceph CLI.
+
+## CSI Driver Options
+
+The CSI driver options mentioned here are applied per Ceph cluster. The following options are available:
+
+* `readAffinity`: Refer to the read affinity section in [ceph-csi-drivers.md](../../Storage-Configuration/Ceph-CSI/ceph-csi-drivers.md#enable-read-affinity-for-rbd-and-cephfs-volumes) for more details.
```

Comment:
In the text link we want to use the normal text, instead of a filename. Also, a brief explanation is helpful.
```suggestion
* `readAffinity`: RBD volumes allow serving reads from an OSD in proximity to the client. Refer to the read affinity section in the [Ceph CSI Drivers](../../Storage-Configuration/Ceph-CSI/ceph-csi-drivers.md#enable-read-affinity-for-rbd-and-cephfs-volumes) doc for more details.
```